Resources/Prototypes/Entities/Structures/Walls/walls.yml

```
@@ -1106,13 +1106,14 @@
   parent: BaseWall
   id: WallWood
   name: wood wall
+  description: The traditional graytide defense.
```

```suggestion
  description: The traditional greytide defense.
```
I did not look at the rest of the code yet.